exercises/concept/cars-assemble/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+pub fn production_rate_per_hour(_speed: u8) -> f32 {
+    unimplemented!("Implement production_rate_per_hour")
+}
+
+pub fn working_items_per_minute(_speed: u8) -> u8 {
```

Comment:
```suggestion
pub fn working_items_per_minute(_speed: i32) -> f64 {
```

Returning a `f64` will allow us to avoid talking about type casting with `as` entirely in this exercise, which would be consistent with the book.